_entities.md

```diff
@@ -242,7 +280,9 @@ Property   | Description
 amount     | The amount to be transacted.
 commission | The total commission taken on this transaction, either at origin or at destination.
 currency   | The currency in which the amount and commission are expressed. The value is always `USD`.
+fee        | The normalized fee amount.
 rate       | The exchange rate for this pair.
+target     | Can be `origin` or `destination` and determines where the fee was applied.
```

Comment:
Looks like all other entries start with "The foo". This should probably adopt the same format, for consistency.

Reply:
This was extracted from the `### Fees` section, which was already using this description. Maybe we can improve some of these descriptions in the future.

Reply:
Sure. It would be nice to have a list of low-hanging fruit like this to improve in the near future.

Reply:
Great. I've created #160 to track this particular issue, and added a ["bite-size" label](https://github.com/uphold/docs/labels/bite-size) for similar issues. Let me know if you prefer a different name for the label, and please tag any relevant existing issues with it.